bio2zarr/vcf2zarr/icf.py

```diff
@@ -458,6 +490,87 @@ def sanitise_value_int_2d(buff, j, value):
         buff[j, :, : value.shape[1]] = value
 
 
+def compute_laa_field(variant) -> np.ndarray:
```

Comment:
Would be good to add a quick summary of what we're doing here, as the process is somewhat involved (necessarily). So, we're talking the local alleles to be anything that's in the genotypes, or has an allele depth of > 0, or is referenced in the PL field?

Reply:
That's correct. I suppose that ideally, the code would infer which alleles are observed based on all the fields that can be localized. If we wanted to start localizing another genotype field in the future, we would need to update `compute_laa_field` to make sure that the LAA value contains all the alleles used in the genotype field.